### PR TITLE
updating jquery version to reflect improvements

### PR DESCRIPTION
**Pull request by Rian501**
Because the React migration is taking some time, and it would be good to get a working testable prototype for farmers to try out, I'm filling in some missing details in the jQuery version of the app.
On this PR: 
The glean request button hides until a person is logged in or registered. (#19, #21) and once they do log in, the register and log in links disappear from the nav (#15).

Log out functionality works and is now linked to the nav buttons. (#8, #21, #15).

Also, I've added the checkbox for is_organic as per #13. (More additions needed)

To Test:
1. Fetch the branch and pull it down.
1. Check the links available before signing in, and confirm what is available after signing in (or signing up).
1. Ensure that when logged in, the log out and glean submission buttons are available.
1. Click log out and ensure the log in and register buttons are now available and that the glean submit and log out buttons are gone. (#8, #15, #21)
1. register a new account and check the console log to see that there is an object reflecting the is_organic boolean. (If you checked the box, it should say true, if not, false)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,9 +22,10 @@
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
       <div class="navbar-nav">
         <a class="nav-item nav-link active" id="log-in-nav" href="#">Log In</a>
+        <a class="nav-item nav-link hidden" id="log-out-nav" href="#">Log Out</a>
         <a class="nav-item nav-link" id="sign-up-nav" href="#">Register</a>
         <!-- <a class="nav-item nav-link" href="#">Profile</a> -->
-        <a class="nav-item nav-link" id="glean-req-nav" href="#">Gleaning Request</a>
+        <a class="nav-item nav-link hidden" id="glean-req-nav" href="#">Gleaning Request</a>
         <a class="nav-item nav-link" target="_blank" href="http://endhunger.org/">Society of St. Andrew</a>
```
Comment (Shu52) [inline index.html]: Please make link https instead of http. Http redirects to https by default

Comment (Shu52) [inline index.html]: add s to http

File: index.html [hunk 2]
```diff
@@ -22,9 +22,10 @@
     <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
       <div class="navbar-nav">
         <a class="nav-item nav-link active" id="log-in-nav" href="#">Log In</a>
+        <a class="nav-item nav-link hidden" id="log-out-nav" href="#">Log Out</a>
```
Comment (Shu52) [inline index.html]: "Log in" remains while a new user is logged in. No, log out link. No login or log out link if an existing user logs in.

File: index.html [hunk 3]
```diff
@@ -22,9 +22,10 @@
...
         <a class="nav-item nav-link" id="sign-up-nav" href="#">Register</a>
         <!-- <a class="nav-item nav-link" href="#">Profile</a> -->
-        <a class="nav-item nav-link" id="glean-req-nav" href="#">Gleaning Request</a>
+        <a class="nav-item nav-link hidden" id="glean-req-nav" href="#">Gleaning Request</a>
```
Comment (Shu52) [inline index.html]: This link is only viewable if you logged in as an existing user. Not viewable after a new user registration

File: index.html (hunk 1 above)
Comment (Shu52) [reply index.html]: Sorry, I was clear in the previous comment. http://endhunger.org/ redirects to https://endhunger.org/

File: index.html (hunk 2 above)
Comment (Shu52) [reply index.html]: My fault i was one commit behind on your branch

**Comment by Shu52** [comment]
no console log when clicking organic checkbox

**Comment by Shu52** [comment]
This a tag
``<a href="/">Glean Tennessee</a>``
is on both registration form page and gleaning request page, and it doesn't seem to serve a purpose

File: index.html (hunk 3 above)
Comment (Rian501) [reply index.html]: Good catch. I believe it is working now, but please test again.

**Comment by Rian501** [comment]
I have also taken out that h1 tag `<a href="/">Glean Tennessee</a>` because I think it looks better on mobile without that anyway!

Thanks.

**Pull request merged**